build.gradle

```diff
@@ -91,26 +88,10 @@ dependencies {
     testImplementation gradleTestKit()
 }
 
-// Exclude transitive slf4j-api dependency from being embedded into the shadow jar
-configurations.runtimeClasspath {
-    exclude group: 'org.slf4j', module: 'slf4j-api'
-}
-
 tasks.withType(Test) {
     useJUnitPlatform()
 }
 
-task relocateShadowJar(type: ConfigureShadowRelocation) {
-    target = tasks.shadowJar
-    prefix = "axion"
-}
-
-tasks.shadowJar.dependsOn tasks.relocateShadowJar
-
-shadowJar {
-    archiveClassifier.set('')
-}
-
 components.java.withVariantsFromConfiguration(configurations.runtimeElements) {
```

Comment:
This section is only for shadow, removing dependencies from POM for publishing (as they are included in the shadow jar).  Should be removed to publish a full POM with dependencies.

Reply:
yes, I've just tested this pr and found this out ;)